.github/workflows/publish.yml

```diff
@@ -0,0 +1,49 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Publish to GitHub
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the develop branch
+on:
+  push:
+    branches: [ 'develop' ]
+  pull_request:
+    branches: [ 'develop' ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - name: "checkout source"
+      uses: actions/checkout@v2
+    - name: "Set up JDK 1.8"
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: "set up node.js"
+      uses: actions/setup-node@v1
+      with:
+        node-version: "12.x"
+        registry-url: https://npm.pkg.github.com/
+        scope: "@useoptic"
+    - name: "intial yarn install"
```

Comment:
Is there any reason it has to be done before bumping the versions?

Reply:
`fs-events` required for your script